wagtail/sites/migrations/0001_initial.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 2.1.3 on 2019-08-19 21:07
+
+from django.db import migrations
+import swapper
+
+
+def initial_site(apps, schema_editor):
+    ContentType = apps.get_model("contenttypes.ContentType")
+    Page = apps.get_model("wagtailcore.Page")
+    Site = apps.get_model(swapper.get_model_name("wagtailcore", "site"))
+
+    page_content_type, created = ContentType.objects.get_or_create(
+        model="page", app_label="wagtailcore"
+    )
+
+    # Get homepage
+    homepage = Page.objects.get(
+        title="Welcome to your new Wagtail site!",
+        slug="home",
+        content_type=page_content_type,
+        path="00010001",
+        depth=2,
+        numchild=0,
+        url_path="/home/",
+    )
+
+    # Create default site
+    Site.objects.get_or_create(
+        hostname="localhost", root_page_id=homepage.id, is_default_site=True
+    )
+
```

Comment:
This migration might work okay in tests, but I imagine both the `Page.objects.get()` and the `Site.objects.get_or_create()` might be problematic when the migration runs for the first time in an existing project after upgrading.

Reply:
Yeah it might be better to do a check to see if `Site.objects.all().count() > 0` since I think that more accurately captures the idea (create a site if and only if we don't currently have one)